modules/cnf-image-based-upgrade-seed-image-guide.adoc

```diff
@@ -4,7 +4,7 @@
 [id="cnf-image-based-upgrade-seed-image-guide_{context}"]
 = Seed image guidelines
 
-The seed image targets a set of {sno} clusters with similar configuration.
+The seed image targets a set of {sno} clusters with the same hardware and similar configuration.
```

Comment:
I would like to update customer document with  below system configuration.

`The CPU topology must align with the seed SNO.
Same number of cores.
Same performance tuning (i.e., reserved CPUs).`

Ref:- https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/seed-image-generation.md#seed-sno-pre-requisites